components/form/birthdayinput.tsx

```diff
@@ -0,0 +1,90 @@
+import FormGroup from './form-group';
+import Hint from './hint';
+import Label from './label';
+import { Field, FieldInputProps, FieldMetaProps } from 'formik';
+import ErrorMessage from './error-message';
+
+
+export function BirthdayInputFields({
```

Comment:
Would probably use DateInput rather than Birthday, which is for a specific use case.